orderer/common/follower/follower_chain.go

```diff
@@ -313,7 +329,7 @@ func (c *Chain) pull() error {
 			// Trigger creation of a new consensus.Chain.
 			c.logger.Info("On-boarding finished successfully, invoking chain creation callback")
```

Comment:
Nit: shouldn't we update this log message to not reference the chain creation callback?

Reply:
done